integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -270,9 +270,9 @@ spec:
         - name: {{ .alias | lower }}
           emptyDir:
             medium: "Memory"
-                  {{- if .quota }} 
+                  {{- if .quota }}
```

Comment:
These 2 trailing whitespace are removed so extra whitespace in templates are removed.